docs/general/information-schema.rst

```diff
@@ -288,24 +288,25 @@ table::
     +-------------------+--------------------------------+-----+--------------------------+
     | locations         | date                           |   3 | timestamp with time zone |
     | locations         | description                    |   6 | text                     |
-    | locations         | id                             |   1 | text                     |
+    | locations         | id                             |   1 | integer                  |
     | locations         | information                    |  11 | object_array             |
     | locations         | information['evolution_level'] |  12 | smallint                 |
     | locations         | information['population']      |  13 | bigint                   |
+    | locations         | inhabitants                    |   7 | object                   |
+    | locations         | inhabitants['description']     |   8 | text                     |
+    | locations         | inhabitants['interests']       |   9 | text_array               |
+    | locations         | inhabitants['name']            |  10 | text                     |
     | locations         | kind                           |   4 | text                     |
+    | locations         | landmarks                      |  14 | text_array               |
     | locations         | name                           |   2 | text                     |
     | locations         | position                       |   5 | integer                  |
-    | locations         | race                           |   7 | object                   |
-    | locations         | race['description']            |   8 | text                     |
-    | locations         | race['interests']              |   9 | text_array               |
-    | locations         | race['name']                   |  10 | text                     |
     | partitioned_table | date                           |   3 | timestamp with time zone |
     | partitioned_table | id                             |   1 | bigint                   |
     | partitioned_table | title                          |   2 | text                     |
     | quotes            | id                             |   1 | integer                  |
     | quotes            | quote                          |   2 | text                     |
     +-------------------+--------------------------------+-----+--------------------------+
-    SELECT 18 rows in set (... sec)
+    SELECT 19 rows in set (... sec)
 
 You can even query this tables' own columns (attention: this might lead to
```

Comment:
lint: should be "table's"

Reply:
fixed in https://github.com/crate/crate/pull/10866/commits/3551afc5a97b5b50159d5257da41be7d9b075aa6